flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/HiveParser.java

```diff
@@ -281,14 +420,542 @@ private void clearSessionState(HiveConf hiveConf) {
         }
     }
 
+    private CatalogSinkModifyOperation createInsertOperation(
```

Comment:
nit: would be better to move the conversions into another class or util, the current `HiveParser` looks quite big now.